src/section_info/mod.rs

```diff
@@ -22,10 +22,7 @@ use xor_name::{Prefix, XorName};
 #[derive(Debug, Serialize, Deserialize, PartialEq, Clone)]
 pub enum Message {
     /// Message to request information about the section that matches the given name.
-    GetSectionQuery {
-        public_key: PublicKey,
-        is_node: bool,
```

Comment:
IIRC this was recently added by @lionel1704 for a fix on bootstrapping of adults.